### PR TITLE
fix: make sure that we get resources only from single site

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
See ticket for a description and video fo the problem

## Solution
1. add a `where` clause on the query for the `siteId`.

### CODE REVIEW

File: apps/studio/src/server/modules/resource/resource.router.ts [hunk 1]
```diff
@@ -33,11 +33,12 @@ export const resourceRouter = router({
     }),
   getFolderChildrenOf: protectedProcedure
     .input(getChildrenSchema)
-    .query(async ({ input: { resourceId, cursor: offset, limit } }) => {
+    .query(async ({ input: { siteId, resourceId, cursor: offset, limit } }) => {
```
Comment (karrui) [inline apps/studio/src/server/modules/resource/resource.router.ts:36]: do we need to update the callsites? or we have always been passing it in but not using it lol!

**Review by karrui — APPROVED** [review]
lgtm

File: apps/studio/src/server/modules/resource/resource.router.ts (hunk 1 above)
Comment (seaerchin) [reply apps/studio/src/server/modules/resource/resource.router.ts:36]: passing in but ont used HAHAHAH

Comment (seaerchin) [reply apps/studio/src/server/modules/resource/resource.router.ts:36]: image

in `MoveResourceModal`, only place where `getFolderChildrenOf` is used

**Pull request merged**